docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -103,25 +121,31 @@ jobs:
 
 ```yaml
 semgrep:
+  # A Docker image with Semgrep installed.
   image: returntocorp/semgrep
+  # Run the "semgrep ci" command on the command line of the docker image.
   script: semgrep ci
 
   rules:
-  # Scan changed files in MRs, block on new issues only (existing issues ignored)
+  # Determine when you want Semgrep to scan your code.
+  # Use Option 1, 2, or both.
+  #
+  # Option 1: Scan changed files in MRs, only report new findings (existing
+  # findings ignored).
   - if: $CI_MERGE_REQUEST_IID
-  # Scan all files on default branch, block on any issues
+  # Option 2: Scan all files on the default branch, report any findings.
   # - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
```

Comment:
Would it be possible to get CSE help here on if we can do scheduled scans (similar to Github Actions setup)? This could happen later so not blocking this PR

Reply:
on it.

Reply:
updated comments. :-)